test/project_v2/main.go

```diff
@@ -37,6 +37,7 @@ var (
 
 func init() {
 
+	crewv1.AddToScheme(scheme)
 	crewv1.AddToScheme(scheme)
```

Comment:
This is a bug where `crewv1.AddToScheme` is getting duplicated. Will fix this in a separate PR.